.goreleaser.yml

```
@@ -37,7 +37,7 @@ universal_binaries:
         - cmd: ./tools/notarize "{{ .Path }}" "acorn-v{{ .Version }}-macOS-universal"
           output: true
           env:
-            - NOTARIZE={{ if index .Env "NOTARIZE" }}{{ .Env.NOTARIZE }}{{ end }}
+            - NOTARIZE={{ if index .Env "NOTARIZE" }}0{{ end }} # Always disable Notarization since it is not currently necessary.
```

not 100% sure but I think just this would work:

```suggestion
            - NOTARIZE=0 # Always disable Notarization since it is not currently necessary.
```

---

Yeah, that'll work. The main difference here is that you can't pass and environment variable to the notarize script but in this context that's fine.